src/sass/extends/_person-profile.scss

```diff
@@ -0,0 +1,71 @@
+/*--------------------------------------------------------------
+Desktop styles
+--------------------------------------------------------------*/
+.uds-person-profile {
+  display: flex;
+  max-width: 800px;
```

Comment:
This is a new addition to the BS4 library and would be a handy block to be able to add to our collection. This section of the SASS expresses a "person" profile layout. 

Link: https://unity.web.asu.edu/@asu-design-system/bootstrap4-theme/index.html?path=/story/components-person-profile--person-profile